src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -234,6 +234,8 @@ void EquationTextBox::OnRemoveButtonClicked(Object ^ sender, RoutedEventArgs ^ e
         m_equationButton->IsChecked = false;
     }
 
+    TraceLogger::GetInstance()->LogGraphButtonClicked(L"RemoveFunction");
```

Comment:
Since we attempt to perform analysis every time the analysis button is clicked, I decided only to log the analysis since we can get a count to determine the number of times the button is clicked.